modules/k3s/src/test/java/org/testcontainers/k3s/KubectlContainerTest.java

```diff
@@ -0,0 +1,66 @@
+package org.testcontainers.k3s;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.WaitingConsumer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.concurrent.TimeUnit;
+
+import static org.testcontainers.containers.Network.newNetwork;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDOUT;
+
+@Slf4j
+public class KubectlContainerTest {
+
+    public static Network network = newNetwork();
+
+    public static K3sContainer k3s = new K3sContainer(DockerImageName.parse("rancher/k3s:v1.21.3-k3s1"))
+        .withNetwork(network)
+        .withNetworkAliases("k3s");
+
+    @BeforeClass
+    public static void setup() {
+        k3s.start();
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        k3s.stop();
+    }
+
+    @Test
+    public void shouldExposeKubeConfigForNetworkAlias() throws Exception {
```

Comment:
We can dramatically simplify this test if we set `command` to something like `top`/`sleep infinity` and then use `execInContainer` once container is started (will give access to standard output as the result)

Reply:
I followed the suggestion of @rnorth  with `OneShotStartupCheckStrategy` to simplify the test code.

Reply:
added `withCopyToContainer(Transferable, String)` usage :) Will merge once CI is green